Sources/Extensions/Widgets/Common/WidgetBasicContainerView.swift

```diff
@@ -170,4 +174,14 @@ struct WidgetBasicContainerView: View {
         @unknown default: return 8
         }
     }
+    
+    private static var clearFamilies: [WidgetFamily] {
```

Comment:
This variable is not very descriptive in my opinion could you try to rename or make a comment explaining it's purpose?